src/modules/Web3Mail/schemas/accepted-proposal-model.ts

```diff
@@ -0,0 +1,17 @@
+import { Schema, model, models } from 'mongoose';
+
+const acceptedProposalEmailSchema = new Schema({
+  id: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  date: {
+    type: Number,
+    required: true,
+    unique: false,
+  },
+});
```

Comment:
use enum for type: 
const testSchema = new mongoose.Schema({
  status: {
    type: String,
    enum: ['valid', 'invalid']
  }
})